docs/en/reference/working-with-associations.rst

```diff
@@ -725,3 +725,26 @@ methods:
     There is a limitation on the compatibility of Criteria comparisons.
     You have to use scalar values only as the value in a comparison or
     the behaviour between different backends is not the same.
+
+
+.. note::
+
+   To have an **is not null** that acts like a *Criteria::expr()->isNotNull('field')* you can use
```

Comment:
I think you miss 2 words here:

To have an **is not null** $word1 that acts like a *Criteria::expr()->isNotNull('field')* $word2 you can use

Also, a colon at the end would be nice IMO

Reply:
Also, please use double backticks around pieces of code